tests/shared/test_data.py

```diff
@@ -82,7 +81,15 @@ def test_default_story_files_are_story_files():
 
 
 def test_default_conversation_tests_are_conversation_tests_yml(tmpdir: Path):
-    parent = tmpdir / DEFAULT_E2E_TESTS_PATH
+    e2e_path = tmpdir / "test_stories.yml"
+    e2e_story = """stories:"""
+    write_text_file(e2e_story, e2e_path)
+
+    assert rasa.shared.data.is_test_stories_file(str(e2e_path))
+
+
+def test_conversation_tests_in_a_directory(tmpdir: Path):
```

Comment:
I'd use `tmp_path`. `tmpdir` is a different type and not of type `Path`
```suggestion
def test_conversation_tests_in_a_directory(tmp_path: Path):
```